openedx/core/djangoapps/course_groups/tests/test_cohorts.py

```diff
@@ -685,12 +685,43 @@ def test_add_user_to_cohort(self, mock_tracker):
             lambda: cohorts.add_user_to_cohort(first_cohort, "non_existent_username")
         )
 
+    def test_get_course_cohort_settings(self):
+        """
+        Test that cohorts.get_course_cohort_settings is working as expected.
+        """
+        course = modulestore().get_course(self.toy_course_key)
```

Comment:
It isn't clear to me if this test is verifying the default settings, or is verifying migrated settings from the toy course. I think there should be three tests:
 - verify the default settings if none are found
 - verify migrated settings from an old course
 - verify new-style settings